### PR TITLE
Update automatic-instrumentation.mdx

### PR DESCRIPTION
**Pull request by lizokm**
Add note with link to docs for users of React.

### CODE REVIEW

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 1]
```diff
@@ -80,6 +80,12 @@ You will need to configure your web server [CORS](https://developer.mozilla.org/
 
 <PlatformContent includePath="performance/beforeNavigate-example" />
 
+<Note>
+  If you're using React, read the docs about how to set up your [React Router
+  integration]
+  (/platforms/javascript/guides/react/configuration/integrations/react-router/).
```
Comment (vivianyentran) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:86]: I think the link text needs to all be on the same line to render properly

**Review by vivianyentran — CHANGES_REQUESTED** [review]

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 2]
```diff
@@ -80,6 +80,12 @@ You will need to configure your web server [CORS](https://developer.mozilla.org/
...
+</Note>
```
Comment (lizokm) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:87]:
```suggestion
<Note>
  If you're using React, read the docs about how to set up your [React Router
  integration](/platforms/javascript/guides/react/configuration/integrations/react-router/).
</Note>
```

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx (hunk 1 above)
Comment (lizokm) [reply src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx]: Yup, you're totally right. I had it on the same line. Not sure why it's doing this...

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 3]
```diff
@@ -80,6 +80,11 @@ You will need to configure your web server [CORS](https://developer.mozilla.org/
 
 <PlatformContent includePath="performance/beforeNavigate-example" />
 
+<Note>
+  If you're using React, read the docs about how to set up your [React Router
+  integration](/platforms/javascript/guides/react/configuration/integrations/react-router/).
+</Note>
+
```
Comment (lizokm) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:87]:
```suggestion
<Note>
  If you're using React, read the docs about how to set up your <a class href=“/platforms/javascript/guides/react/configuration/integrations/react-router/”>React Router
  integration</a>.
</Note>

```

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 4]
```diff
@@ -80,6 +80,11 @@ You will need to configure your web server [CORS](https://developer.mozilla.org/
 
 <PlatformContent includePath="performance/beforeNavigate-example" />
 
+<Note>
+  If you're using React, read the docs about how to set up your <a class href=“/platforms/javascript/guides/react/configuration/integrations/react-router/”>React Router
+  integration</a>.
+</Note>
```
Comment (lizokm) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:86]:
```suggestion
<Note>
 
 If you're using React, read our docs about how to set up your <a class href=“/platforms/javascript/guides/react/configuration/integrations/react-router/”>React Router
  integration</a>.
  
</Note>
```

File: src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx [hunk 5]
```diff
@@ -80,6 +80,13 @@ You will need to configure your web server [CORS](https://developer.mozilla.org/
 
 <PlatformContent includePath="performance/beforeNavigate-example" />
 
+<Note>
+ 
+ If you're using React, read our docs about how to set up your <a class href=“/platforms/javascript/guides/react/configuration/integrations/react-router/”>React Router
+  integration</a>.
+
```
Comment (lizokm) [inline src/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx:87]:
```suggestion
<Note>
 
 If you're using React, read our docs about how to set up your <a class href="/platforms/javascript/guides/react/configuration/integrations/react-router/">React Router
  integration</a>.
  
```

**Review by vivianyentran — APPROVED** [review]

**Pull request merged**